File: Java/715.java
import java.util.Scanner;

/**
 *
 * 715 - ¿Hay Suficientes? - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int pruebas = sc.nextInt();
        for (int i = 1; i<=pruebas; i++) {
            int uvas = sc.nextInt();
            int personas = sc.nextInt();
            if (uvas / personas >= 12) {
                System.out.println("SI");
            } else {
                System.out.println("NO");
            }
        }
    }

}

File: C/715.c
#include <stdio.h>
#include <stdlib.h>

// 715 - ¿Hay Suficientes? - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop;
    scanf("%d", &loop);
    int i;
    int grapes = 0;
    int people = 0;
    for (i = 0; i<loop; i++) {
        scanf("%d", &grapes);
        scanf("%d", &people);
        if (grapes/people>=12) {
            printf("SI\n");
        } else {
            printf("NO\n");
        }
    }
    return 0;
}
